ui/e2e/src/types/happo.d.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
The one down side of happo is they're a small enough startup that not everything has types yet. Thankfully, there's a very small surface area we have to touch, so it was easy to add some internally. It might be nice to contribute to [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) at some point, but I don't think it's worth the overhead right now.